app/packs/src/components/ketcher-templates/CommonTemplatesList.js

```diff
@@ -63,37 +61,33 @@ const CommonTemplatesList = memo(({ options, onClickHandle, selectedItem }) => {
         </div>
       </div>
 
-      <Modal show={commonTemplateModal} onHide={() => setCommonTemplateModal(false)}>
-        <Modal.Header closeButton >
+      <Modal
+        centered
+        show={commonTemplateModal}
+        onHide={() => setCommonTemplateModal(false)}
+      >
+        <Modal.Header closeButton>
           {toolTip}
         </Modal.Header>
         <Modal.Body>
-          <Panel style={{ height: 300 }}>
-            <Panel.Heading>
-              <Panel.Title>
-                Common Template list:
-              </Panel.Title>
-            </Panel.Heading>
-            <Panel.Body style={{ height: '80%', overflow: 'auto' }}>
-              <div>
-                <input
-                  type="text"
-                  placeholder="Search templates by Name..."
-                  value={searchQuery}
-                  onChange={(e) => setSearchQuery(e.target.value)}
-                  className='common-template-search'
-                />
-                {filteredOptions.length != options.length &&
-                  <div style={{ marginVertical: '10px' }}>
-                    {filteredOptions.length} out of {options.length} templates found
-                  </div>
-                }
-                {filteredOptions.map((item, idx) => (
-                  <CommonTemplateItem key={idx} item={item} onClickItem={(value) => onSelectItem(value)} />
-                ))}
-              </div>
-            </Panel.Body>
-          </Panel>
+          <Form.Control
+            type="text"
+            placeholder="Search templates by Name..."
+            value={searchQuery}
+            onChange={(e) => setSearchQuery(e.target.value)}
+          />
+          {filteredOptions.length != options.length && (
+            <div style={{ marginVertical: '10px' }}>
```

Comment:
Could this marginVertical be changed to my-?

Reply:
Especially since `marginVertical` does not even exist… 😅